airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/source/relationaldb/state/SourceStateIterator.kt

```diff
@@ -84,7 +87,7 @@ open class SourceStateIterator<T>(
      * re-enable it once we do the work for project Counts: Emit Counts in Full Refresh
      */
     private fun shouldAttachCountWithState(): Boolean {
-        return stream?.syncMode != SyncMode.FULL_REFRESH
+        return true
```

Comment:
does this mean we no longer need this function?

Reply:
+1 this should be removed here and below